### DISCUSSION TITLE
C# with UWP (Universal Windows Platform) using XAML learning curve

### DISCUSSION THREAD

**Discussion by tafemq** [tafecd/Tafe230120#6] [General]
I am totally new to VS 2019 and the XAML UWP design/coding.
I was thinking about normal form .cs, never xaml.cs and it was very hard to get it started and then working with the combo boxes values etc. I am used to MS Access forms or Excel where dealing with controls seems more natural.
How did you learn VS Studio, which version and what was your training with XAML (UWP)?

**Comment by tafecd** [comment]
I started my Cert 4 back in 2021 and did 4BUI (XAML) and 4C#B (C# coding). Both were very interesting and definitely helped with the coding portion of this assignment.

**Reply by tafemq** [reply]
@tafecd You did help me with the start selecting the correct project XAML template. But I have completed my currency conversion calculator with the help of Google when available and my programming life experience. Does it mean that the programming classes could become obsolete and attended mainly to receive a certification that an autodidact could not get?
